frontend/src/main/pages/ModeratePage.jsx

```diff
@@ -14,20 +17,64 @@ const Moderate = () => {
   } = useBackend(
     // Stryker disable next-line all : don't test internal caching of React Query
     ["/api/reviews/needsmoderation"],
+    // Stryker disable next-line all : don't test internal caching of React Query
     { method: "GET", url: "/api/reviews/needsmoderation" },
     // Stryker disable next-line all : don't test internal caching of React Query
     [],
   );
 
+  //
+  // Fetch aliases awaiting approval
+  //
+  const { data: aliases } = useBackend(
+    // Stryker disable next-line all : don't test internal caching of React Query
+    ["/api/admin/users/needsmoderation"],
+    // Stryker disable next-line all : don't test internal caching of React Query
+    { method: "GET", url: "/api/admin/users/needsmoderation" },
+    // Stryker disable next-line all : don't test internal caching of React Query
+    [],
+  );
+
   const moderatorOptions =
     hasRole(currentUser, "ROLE_ADMIN") ||
     hasRole(currentUser, "ROLE_MODERATOR");
 
+  const approveCallback = async (alias) => {
+    const user = alias.row.original;
+
+    await axios.put("/api/currentUser/updateAliasModeration", null, {
+      params: {
+        id: user.id,
+        approved: true,
+      },
+    });
+    // Stryker disable next-line all
+    toast(`Approved alias for ${user.email}`);
+  };
+
+  const rejectCallback = async (alias) => {
+    const user = alias.row.original;
+
+    await axios.put("/api/currentUser/updateAliasModeration", null, {
+      params: {
+        id: user.id,
+        approved: false,
+      },
+    });
+    // Stryker disable next-line all
+    toast(`Approved alias for ${user.email}`);
+  };
```

Comment:
Same comments as above, except the toast message (if you keep it) should say "Rejected" not "Approved".